app/resources/views/pages/location_management/suggestion_detail.blade.php

```diff
@@ -0,0 +1,57 @@
+@extends('layouts.default')
+@section('head-content')
+    <script src="/js/suggestion_detail.js"></script>
```

Comment:
Swap the script elements for including suggestion_detail.js with jquery so $ is defined before you run suggestion_detail.js.